src/command_modules/azure-cli-network/azure/cli/command_modules/network/_help.py

```diff
@@ -1159,10 +1159,38 @@
     type: group
     short-summary: Manage Network Security Groups (NSG)
 """
+
 helps['network nsg rule'] = """
     type: group
     short-summary: Manage NSG rules
 """
+
+helps['network nsg rule create'] = """
+    type: command
+    short-summary: Create a new NSG rule.
+"""
+
+helps['network nsg rule delete'] = """
+    type: command
+    short-summary: Delete an NSG rule.
```

Comment:
No. If you read NSG as "En-es-gee" instead of "network security group" then an is the correct word :)